apis/logging/v1/elasticsearch_types.go

```diff
@@ -23,7 +23,8 @@ const (
 // +kubebuilder:printcolumn:name="Shard Allocation",JSONPath=".status.shardAllocationEnabled",type=string
 // +kubebuilder:printcolumn:name="Index Management",JSONPath=".status.indexManagement.State",type=string
 //
-// Elasticsearch is the Schema for the elasticsearches API
+// An Elasticsearch cluster instance
+// +operator-sdk:csv:customresourcedefinitions:displayName="Elasticsearch",resources={{Pod,v1},{Deployment,v1},{StatefulSet,v1},{ReplicaSet,v1},{ConfigMap,v1},{Service,v1},{Route,v1}}
```

Comment:
Looking through this list: https://github.com/openshift/elasticsearch-operator/blob/master/pkg/k8shandler/reconciler.go#L38

Tells us, that Elasticsearch CR is also creating the following missing resources:
- CronJob
- PrometheusRule
- Role
- RoleBinding
- ServiceAccount
- ServiceMonitor

According to OLM documentation these resources are only descriptors to inform users on which resources are owned by a CRD:
```
Resources: Your CRDs will own one or more types of Kubernetes objects. These are listed in the resources section to inform your end-users of the objects they might need to troubleshoot or how to connect to the application, such as the Service or Ingress rule that exposes a database.
```
See https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#owned-crds

I suggest we complete this list then.